cmd/operator/main.go

```diff
@@ -245,6 +246,15 @@ func main() {
 		os.Exit(1)
 	}
 
+	// If proxy is enabled, disabled, or edited during WMCO runtime, the WMCO pod will be restarted by OLM. This could
+	// happen in the middle of node configuration, at which the controllers will reconcile once the WMCO pod restarts
+	if proxyEnabled {
+		if err := configMapReconciler.EnsureTrustedCAConfigMapExists(); err != nil {
```

Comment:
This can be created as part of the operator manifests but we lose the ability to kick back changes (if a user deleted the trusted-ca CM or removed its expected injection label). If we are fine with that, I can updated the PR to go the operator manifest route

Reply:
> Can this be created as part of the operator manifests instead?

resolution: we decided against this for the above reason